src/ImageSharp.Formats.Png/PngEncoderCore.cs

```diff
@@ -495,7 +495,7 @@ private QuantizedImage<TColor> WritePaletteChunk<TColor>(Stream stream, PngHeade
 
             if (this.Quantizer == null)
             {
-                this.Quantizer = new WuQuantizer<TColor>();
+                this.Quantizer = new OctreeQuantizer<TColor>();
```

Comment:
Hmm, this has nothing to do with the Dithering. 
I think we need a benchmark measuring the perf effects of switching quantizers. OctreeQuantizer seemed to be really slow when I tested it with Gif.
Certainly a winner in quality though.